assets/src/models/vehicleData.ts

```diff
@@ -130,7 +130,7 @@ const baseVehicleData = {
   end_of_trip_type: EndOfTripTypeData,
   block_waivers: array(BlockWaiverData),
   crowding: nullable(CrowdingData),
-  state_of_charge: optional(StateOfCharge),
+  state_of_charge: optional(nullable(StateOfCharge)),
```

Comment:
question: I would have though that this would be the same as `crowding` on line 132. Can you say more about why this one needs to be both nullable and optional?